components/main/blocks/BlockContentRenderer.tsx

```diff
@@ -486,7 +486,7 @@ export const SectionBlockContentRenderer: React.FC<{ blocks: any }> = ({ blocks
             );
           case "dkmembershipwidget":
             return (
-              <DKMembershipWidget key={block._key || block._id} config={block.confituration} />
+              <DKMembershipWidget key={block._key || block._id} config={block.configuration} />
```

Comment:
Lol.
But this change also requires us to update the tests because prior to this, the content of the widget was from the `defaultConfig` rather than the Sanity config.
Now that it's pulling from the Sanity config, the content is different and so the tests were failing.